index.html

```diff
@@ -1007,17 +942,6 @@ <h4>
         devices, such as writable tags, when they come in range, as
         <a>NDEF message</a>s.
       </li>
-      <li>
-        [future] Allow manual connection for various technologies such as
-        NFC-A and NFC-F depending on the secondary device.
```

Comment:
So there were removed but not mentioned together with ISO-DEP... Also should such be links? I think we mention these while discussion the tag technologies

Reply:
I think adding is above (as you did) is fine

Reply:
I think we could just leave this removed in this place. The "Allow manual connection for ..." sentence could be moved to the use-cases doc (probably already there).